docs/resources/sonarqube_project.md

```diff
@@ -29,7 +29,8 @@ The following arguments are supported:
 
 - name - (Required) The name of the Project to create
 - project - (Required) Key of the project. Maximum length 400. All letters, digits, dash, underscore, period or colon.
-- visibility - (Optional) Whether the created project should be visible to everyone, or only specific user/groups. If no visibility is specified, the default project visibility of the organization will be used.
+- visibility - (Optional) Whether the created project should be visible to everyone, or only specific user/groups. If no visibility is specified, the default project visibility of the organization will be used. The only possible values are `public`
```

Comment:
"The only valid values" would probably be more accurate than "possible values" since you can pass in any value you like - it just might not be valid!

Reply:
@freeranger what about now

Reply:
done